cmd/oras/root/pull.go

```diff
@@ -219,14 +228,16 @@ func doPull(ctx context.Context, src oras.ReadOnlyTarget, dst oras.GraphTarget,
 		var ret []ocispec.Descriptor
 		for _, s := range nodes {
 			if s.Annotations[ocispec.AnnotationTitle] == "" {
-				skippedLayers++
+				if s.Digest != ocispec.DescriptorEmptyJSON.Digest {
+					skippedLayers++
+				}
```

Comment:
Suggest revert the change since it may also silently skipped an unnamed layer with content `{}`. We can skip config blob in the above `if config != nil` block.
```suggestion
			skippedLayers++
```